lib/rules/one-var.js

```diff
@@ -272,6 +276,87 @@ module.exports = {
             return true;
         }
 
+        /**
+         * Fixer to join VariableDeclaration's into a single declaration
+         * @param   {VariableDeclaration} declarations The `VariableDeclaration` to join
```

Comment:
I think `declarations` actually has type `VariableDeclarator[]`, not `VariableDeclarations`. Can you update this comment to reflect that?